index.html

```diff
@@ -134,7 +134,7 @@ <h3>Google Summer of Code</h3>
 
     <h3>Outreachy (winter rounds only)</h3>
         <p class="years">
-            <a href="/outreachy/2017.html">2017</a>
+            <a href="/Outreachy/2017.html">2017</a>
```

Comment:
@Saichethan Please create a separate issue and PR requiring this change. Actually there are two different kind of changes in the same PR. Hope you understand!